libs/common/src/blockchain/blockchain.service.ts

```diff
@@ -95,7 +95,7 @@ export class BlockchainService implements OnApplicationBootstrap, OnApplicationS
     return this.api.rpc.chain.getBlockHash(block);
   }
 
-  public getBlock(block: BlockHash): Promise<SignedBlock> {
+  public getBlock(block?: BlockHash): Promise<SignedBlock> {
```

Comment:
Nit: I feel like the check for if the block exists or not should be before we call this function. Seems like every time we call getBlock, we can assume that we have an existing blockhash.

Reply:
That's not the point; if you call with no arguments you get the latest block